index.js

```diff
@@ -579,6 +579,66 @@ module.exports = {
     });
   },
 
+  geoHashAgg: function (table, limit, precision, options, callback) {
+    var self = this;
+
+    var whereFilter, geomFilter;
+    if ( options.where ){
+      if ( options.where != '1=1'){
+        var clause = this.createWhereFromSql(options.where);
+        whereFilter = ' WHERE ' + clause;
+      } else {
+        whereFilter = ' WHERE ' + options.where;
+      } 
+    }
+
+    if ( options.geometry ){
+      var geom = this.parseGeometry( options.geometry );
+
+      if ((geom.xmin || geom.xmin === 0) && (geom.ymin || geom.ymin === 0)){
```

Comment:
@dmfenton ahh so basically geom.xmin or geom.ymin can be zero since they are lat/longs. If they are equal to 0 then this check fails in JS (since zero equates to false). So we need to explicitly check for zero and let those values pass.

Now the real question is why dont we check for xmax or ymax :)